src/autoapp/Service/SensorService.cpp

```diff
@@ -31,8 +31,8 @@ namespace service
 {
 
 SensorService::SensorService(boost::asio::io_service& ioService, aasdk::messenger::IMessenger::Pointer messenger)
-    : strand_(ioService),
-      timer_(ioService),
+    : timer_(ioService),
```

Comment:
any reason this changed? Assuming based on the order they were declared?

Reply:
Constructors call initializers in the order the vars are declared in the header, not the order you call their initializers in the constructor itself.  So if you list the initializers in a different order it can make you think that side effects happen in the order they show up which is not true.  sorting the initializers so that they have the same ordering as the variables in the header is good practice.  GCC emits a warning if you don't.

Reply:
yeah thats what i meant, there was a warning for this? I don't remember seeing it when I would build it

but then again there were so many other warnings I could have just missed it :p

Reply:
Correct.  Lost in the noise.